modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -13,6 +13,12 @@ CKEditorSampleDisplayContext ckEditorSampleDisplayContext = (CKEditorSampleDispl
 List<TabsItem> tabsItems = ckEditorSampleDisplayContext.getTabsItems();
 %>
 
+<portlet:renderURL var="viewXSSURL">
+	<portlet:param name="mvcPath" value="/view_xss.jsp" />
+</portlet:renderURL>
+
+<a href="<%= viewXSSURL %>">Go to XSS View</a>
+
```

Comment:
I realize this is a sample module, but does the solution need to be this ugly?

![image](https://github.com/user-attachments/assets/d90b6162-a18a-442b-af0c-ec19ee13ae6f)

In addition to ugliness, once you go to XSS page there is no way to navigate back.

And the sample has confusing `Before` and `After` paragraphs that serve no purpose. These looks like leftovers from local dev testing.

And the sample is only on legacy module, so this branching on `view` is misleading.

And the new sample does not have unique name, it should not be `sampleLegacyEditor`.
 
With all above in mind, can we have this case on the existing legacy sample? A link within the legacy sample can reload the page with the messed up param.

Reply:
Sure, I just took Dani's code and confirmed with him that it doesn't need any stylistic changes so I just stuck with it. I'll improve it.